src/GovUk.Education.ExploreEducationStatistics.Content.Services/Mappings/MappingProfiles.cs

```diff
@@ -16,15 +16,15 @@ public MappingProfiles()
             CreateContentBlockMap();
 
             CreateMap<ContentSection, ContentSectionViewModel>().ForMember(dest => dest.Content,
-                m => m.MapFrom(section => section.Content.OrderBy(contentBlock => contentBlock.Order)));
+                m => m.MapFrom(section => Enumerable.OrderBy<ContentBlock, int>(section.Content, contentBlock => contentBlock.Order)));
```

Comment:
What's the reason for moving away from the extension method for this and the next two changes in this file?

Reply:
No idea, I didn't actually write this. Must have been part of Rider's refactoring. Will revert it.

Reply:
Ok great! Thanks 🙂